appcues/src/main/java/com/appcues/analytics/AutoPropertyDecorator.kt

```diff
@@ -88,6 +99,11 @@ internal class AutoPropertyDecorator(
             currentScreen = attributes[ActivityRequestBuilder.SCREEN_TITLE_ATTRIBUTE]?.toString()
             sessionPageviews += 1
         } else if (event.name == AnalyticsEvent.SessionStarted.eventName) {
+            // on session start set _device key
+            attributes[DEVICE_PROPERTY] = hashMapOf<String, Any>().apply {
+                putAll(applicationProperties)
```

Comment:
`applicationProperties` includes `_sdkName` but `_sdkName` doesn't appear in the push stuff. Does it matter if we send it? Should we add `_sdkName` to the docs/diagrams?

Reply:
I think it's fine to send, I think it all just gets dumped into a somewhat unstructured dynamo object and we can send whatever props we want.

Generally speaking, my idea was just to include some set of our existing known "app props" totally for reference purposes, to understand what a particular device record in dynamo is really tied back to from the app/os/sdk side.